torchci/test/drci.test.ts

```diff
@@ -308,7 +308,8 @@ describe("Update Dr. CI Bot Unit Tests", () => {
     const { pending, failedJobs } = await updateDrciBot.getWorkflowJobsStatuses(
       pr_1001,
       [],
-      new Map()
+      new Map(),
+      []
```

Comment:
Just curious: wouldn't be better to add default parameter to `getWorkflowJobsStatuses` rather than pass an empty array all the time?